### ISSUE TITLE
Persist annotations when using SQLitedict

### ISSUE THREAD

**Issue by aronasorman** [learningequality/ka-lite#4335]
See https://github.com/learningequality/ka-lite/pull/4306#discussion_r38388364.

**Comment by aronasorman** [comment]
We can reuse the `DO_NOT_RELOAD_CONTENT_CACHE_AT_STARTUP` flag and deprecate the old method of caching startup data.

**Comment by aronasorman** [comment]
Looking at this for a short while. I couldn't run `create_content_db` with the latest `0.15.x`. I get this error:
```
>>> from django.conf import settings
>>> settings.CONTENT_FILEPATH
Traceback (most recent call last):
  File "<console>", line 1, in <module>
  File "/home/aron/src/github.com/aronasorman/ka-lite/python-packages/django/conf/__init__.py", line 54, in __getattr__
    return getattr(self._wrapped, name)
AttributeError: 'Settings' object has no attribute 'CONTENT_FILEPATH'
```

Looking at `topic_tools/settings.py` where `CONTENT_FILEPATH` is defined, it also requires the `settings` module. So there might be some circular import going on here that's preventing this module to load.

**Comment by rtibbles** [comment]
Yeah, I made changes to fix this in the branch I was working on. Still not got to the root of most things in the topic tree seeming available though (even though the individual videos are correctly annotated in the content caches.

**Comment by rtibbles** [comment]
OK, the video annotation issue seems to be extant generally on 0.15.x, investigating.

**Comment by rtibbles** [comment]
Issue here: #4443

**Comment by rtibbles** [comment]
Fixed in #4446
